components/compositing/compositor.rs

```diff
@@ -1518,6 +1527,22 @@ impl IOCompositor {
                     },
                     _ => {},
                 }
+
+                if let Some(_lcp) = self.lcp_detector.calculate_largest_contentful_paint(
```

Comment:
Please don't prefix with `_` if the binding is actually used. 
If `lcp` is only conditionally used (based on a feature flag), you could also allow the unused variable, or "use" it by e.g. dropping it.